articles/quickstart/backend/aspnet-core-webapi/files/configure-middleware.md

```diff
@@ -30,13 +26,8 @@ public void ConfigureServices(IServiceCollection services)
           );
       });
 
-    services.AddSingleton<IAuthorizationHandler, HasScopeHandler>();
-}
+    builder.Services.AddSingleton<IAuthorizationHandler, HasScopeHandler>();
 
-public void Configure(IApplicationBuilder app, IHostingEnvironment env)
-{
-  // Some code omitted for brevity...
-  app.UseAuthentication();
-  app.UseAuthorization();
-}
+app.UseAuthentication();
```

Comment:
Can we add what app is here ?

Reply:
Gotya! All done. Please see https://github.com/auth0/docs/pull/10231/commits/71b753076a1e4d54f2771f9b0e4cda22030575f4. I will let you resolve all conversations yourself to make sure I didn't miss anything. Thanks.